include/eve/detail/hz_device.hpp

```diff
@@ -49,13 +49,13 @@ namespace eve
   {
     if constexpr(eve::scalar_value<R>)
     {
-      if(todo){ r = f(ts...); return false_(as(r));}
-      return false_(as(r));
+      if(todo){ r = f(ts...); return false_(as(todo));}
+      return false_(as(todo));
     }
     else
     {
       if(eve::any(todo))  r = if_else(todo, f(ts...), r);
-      return eve::false_(as(r));
+      return eve::false_(as(todo));
```

Comment:
Why would this change. Seems like R is still here.

Reply:
I have no idea what this function is, if @jfalcou is OK - I'm OK.

Reply:
This is some internal jtlap tools, this change is fine.